yarn.lock

```diff
@@ -1579,9 +1579,9 @@
   integrity sha512-+iTbntw2IZPb/anVDbypzfQa+ay64MW0Zo8aJ8gZPWMMK6/OubMVb6lUPMagqjOPnmtauXnFCACVl3O7ogjeqQ==
 
 "@stencil/core@^2.0.1":
-  version "2.5.2"
-  resolved "https://registry.yarnpkg.com/@stencil/core/-/core-2.5.2.tgz#ad00d495ee37bbed4044524d59c7f22de15ab4a7"
-  integrity sha512-bgjPXkSzzg1WnTgVUm6m5ZzpKt602WmA/QljODAW1xVN40OHJdbGblzF/F6MFzqv2c5Cy30CB41arc8qADIdcQ==
+  version "2.6.0"
```

Comment:
I mean, I'm fine with this, but should this be included? I recommend undoing these changes, updating the `web-components` dependency version in the `package.json` to use the latest tagged version like it should, and `yarn install` again to make sure that the changes to lock file committed here really should be there.